docs/config_options.rst

```diff
@@ -3,43 +3,143 @@ Configuration Options
+
+       .. _config_cache_python_memory_portion:
+   * - ``cache_python_memory_portion`` :ref:`🔗 <config_cache_python_memory_portion>`
+     - If tiles are cached with python, the cache is sized so that it is expected to use less than 1 / (``cache_python_memory_portion``) of the available memory.
+     - ``int``
+     -
+     -
+
+       .. _config_cache_memcached_url:
+   * - ``cache_memcached_url`` :ref:`🔗 <config_cache_memcached_url>`
+     - If tiles are cached in memcached, the url or list of urls where the memcached server is located.
+     - ``str | List[str]``
+     - ``False``
+     - ``"127.0.0.1"``
+
+       .. _config_cache_memcached_username:
+   * - ``cache_memcached_username`` :ref:`🔗 <config_cache_memcached_username>`
+     - A username for the memcached server.
+     -
+     -
+     - ``None``
+
+       .. _config_cache_memcached_password:
+   * - ``cache_memcached_password`` :ref:`🔗 <config_cache_memcached_password>`
+     - A password for the memcached server.
+     -
+     -
+     - ``None``
+
+       .. _config_cache_redis_url:
+   * - ``cache_redis_url`` :ref:`🔗 <config_cache_redis_url>`
+     - If tiles are cached in redis, the url or list of urls where the redis server is located.
+     - ``str | List[str]``
+     -
+     - ``"127.0.0.1:6379"``
+
+       .. _config_cache_redis_username:
+   * - ``cache_redis_username`` :ref:`🔗 <config_cache_redis_username>`
+     - A username for the redis server.
+     -
+     -
+     - ``None``
+
+       .. _config_cache_redis_password:
+   * - ``cache_redis_password`` :ref:`🔗 <config_cache_redis_password>`
+     - A password for the redis server.
+     -
+     -
+     - ``None``
+
+       .. _config_cache_tilesource_memory_portion:
+   * - ``cache_tilesource_memory_portion`` :ref:`🔗 <config_cache_tilesource_memory_portion>`
+     - Tilesources are cached on open so that subsequent accesses can be faster.  These use file handles and memory.  This limits the maximum based on a memory estimation and using no more than 1 / (``cache_tilesource_memory_portion``) of the available memory.
+     - ``int``
+     -
+     -
+
+       .. _config_cache_tilesource_maximum:
+   * - ``cache_tilesource_maximum`` :ref:`🔗 <config_cache_tilesource_maximum>`
+     - If this is non-zero, this further limits the number of tilesources than can be cached to this value.
+     -
+     -
+     -
+
+       .. _config_cache_sources:
+   * - ``cache_sources`` :ref:`🔗 <config_cache_sources>`
+     - If set to False, the default will be to not cache tile sources.  This has substantial performance penalties if sources are used multiple times, so should only be set in singular dynamic environments such as experimental notebooks.
+     -
+     -
+     -
+
+       .. _config_max_small_image_size:
+   * - ``max_small_image_size`` :ref:`🔗 <config_max_small_image_size>`
+     - The PIL tilesource is used for small images if they are no more than this many pixels along their maximum dimension.
+     - ``int``
+     -
+     -
```

Comment:
Default is 4096.  Specifying values greater than this _could_ reduce compatibility with tile use on some browsers.  In general 8192 is safe for all modern systems, and values greater than 16384 should not be specified if the image will be viewed in any browser.